third_party/breakpad.cmake

```diff
@@ -0,0 +1,192 @@
+        "client/linux/log/log.cc"
+        "client/linux/microdump_writer/microdump_writer.cc"
+        "client/linux/minidump_writer/linux_dumper.cc"
+        "client/linux/minidump_writer/linux_ptrace_dumper.cc"
+        "client/linux/minidump_writer/minidump_writer.cc"
+        "client/minidump_file_writer.cc"
+        "common/convert_UTF.c"
+        "common/linux/elfutils.cc"
+        "common/linux/file_id.cc"
+        "common/linux/guid_creator.cc"
+        "common/linux/linux_libc_support.cc"
+        "common/linux/memory_mapped_file.cc"
+        "common/linux/safe_readlink.cc"
+        "common/md5.cc"
+        "common/string_conversion.cc"
+    )
+endif()
+
+# TODO
+# symupload
+# minidump_upload
+
+if(NOT DISABLED_CXX)
+    abs_list(breakpad_srcs ${breakpad_dir})
+    add_library(breakpad STATIC ${breakpad_srcs})
+    target_include_directories(breakpad PUBLIC ${breakpad_dir})
+
+    if(ANDROID)
+        set_property(SOURCE "${breakpad_dir}/common/android/breakpad_getcontext.S" PROPERTY LANGUAGE C)
+        target_sources(breakpad PRIVATE "${breakpad_dir}/common/android/breakpad_getcontext.S")
+        target_include_directories(breakpad SYSTEM BEFORE PRIVATE "${breakpad_dir}/common/android/include")
+        target_compile_definitions(breakpad PUBLIC "__STDC_FORMAT_MACROS")
+    elseif(APPLE)
+        find_library(CARBON_LIBRARY Carbon)
+        find_library(FOUNDATION_LIBRARY Foundation)
+        find_library(SYSTEM_CONFIGURATION_LIBRARY SystemConfiguration)
+        find_library(APPKIT_LIBRARY AppKit)
+        find_library(COCOA_LIBRARY Cocoa)
+
+        set(inspector_srcs
+            "common/convert_UTF.c"
+            "common/mac/arch_utilities.cc"
+            "common/mac/bootstrap_compat.cc"
+            "common/mac/file_id.cc"
+            "common/mac/launch_reporter.cc"
+            "common/mac/macho_id.cc"
+            "common/mac/macho_utilities.cc"
+            "common/mac/macho_walker.cc"
+            "common/mac/MachIPC.mm"
+            "common/mac/string_utilities.cc"
+            "common/md5.cc"
+            "common/string_conversion.cc"
+            "client/mac/crash_generation/ConfigFile.mm"
+            "client/mac/crash_generation/Inspector.mm"
+            "client/mac/crash_generation/InspectorMain.mm"
+            "client/mac/handler/breakpad_nlist_64.cc"
+            "client/mac/handler/dynamic_images.cc"
+            "client/mac/handler/minidump_generator.cc"
+            "client/minidump_file_writer.cc"
+        )
+        abs_list(inspector_srcs ${breakpad_dir})
+        add_executable(inspector ${inspector_srcs})
+        target_include_directories(inspector PUBLIC
+            ${breakpad_dir}
+            "${breakpad_dir}/common/mac"
+            "${breakpad_dir}/client/apple/Framework"
+        )
+        target_compile_options(inspector PRIVATE "-Wno-deprecated")
+        target_link_libraries(inspector ${CARBON_LIBRARY} ${FOUNDATION_LIBRARY})
+
+        # crash_report_sender_resources
```

Comment:
What are these? Are they TODO?

Reply:
actually they were unnecessary, I've removed the crash_report_sender and inspector targets.